internal/instruments.go

```diff
@@ -0,0 +1,38 @@
+package internal
+
+import (
+	"go.opentelemetry.io/otel/api/global"
+	"go.opentelemetry.io/otel/api/metric"
+)
+
+type openTelemetryInstrumentation struct {
+	// Count of write commands performed
+	WriteCount metric.Int64Counter
+	// Count of new connections
+	NewConnectionsCount metric.Int64Counter
+}
+
+var Instruments = initInstruments()
```

Comment:
Can't this be just vars? E.g.

```
var (
    WritesCounter metric.Int64Counter
    ConnsCounter metric.Int64Counter
)
```

Reply:
We thought it might be more readable to use the `Instruments` object to hold the different counters so we can call `internal.Instruments.WriteCount.Add(ctx, 1)` instead of `internal.WriteCount.Add(ctx, 1)`. If you think it'd be better to remove the `Instruments` object, we can definitely remove it. Please let us know!

Reply:
Updated so we use the vars directly as per your suggestion, and revalidated behavior using the example client. Thanks for your feedback!